src/main/java/com/flightstats/hub/webhook/InternalWebhookClient.java

```diff
@@ -45,16 +45,40 @@ boolean stop(String name, String server) {
         return put(server + "/internal/webhook/stop/" + name);
     }
 
-    Optional<String> runOnServerWithFewestWebhooks(String name) {
-        return runOnOneServer(name, getOrderedServers());
+    Optional<String> runOnOnlyOneServer(String name, Collection<String> runningServers) {
```

Comment:
This new method does the following:

1. Tries to confirm the webhook is running/started on one of the nodes in the list
2. Sends stop requests to the remaining nodes that were passed in.
3. If none of the nodes in the list responded that the run is successful, it'll try to fallback to one of the remaining nodes in the cluster that was not in the list.